applications/luci-app-watchcat/luasrc/model/cbi/watchcat/watchcat.lua

```diff
@@ -1,53 +1,123 @@
+                               "suffix <b>m</b> for minutes, <b>h</b> for hours or <b>d</b> " ..
+                               "for days. <br><br>Examples:<ul><li>10 seconds would be: <b>10</b> or <b>10s</b></li><li>5 minutes would be: <b>5m</b></li><li>1 hour would be: <b>1h</b></li><li>1 week would be: <b>7d</b></li><ul>"))
+pingperiod.default = "30s"
+pingperiod:depends({
+    mode = "ping_reboot"
+})
+pingperiod:depends({
+    mode = "restart_iface"
+})
+
+pingsize = s:option(ListValue, "pingsize", translate("Ping Packet Size"),
+               translate(
+                   "<i>Applies to Ping Reboot and Network Restart modes</i>"))
+pingsize:value("small", translate("Small: 1 byte"))
+pingsize:value("windows", translate("Windows: 32 bytes"))
+pingsize:value("standard", translate("Standard: 56 bytes"))
+pingsize:value("big", translate("Big: 248 bytes"))
+pingsize:value("huge", translate("Huge: 1492 bytes"))
+pingsize:value("jumbo", translate("Jumbo: 9000 bytes"))
+pingsize.default = "standard"
+pingsize:depends({
+    mode = "ping_reboot"
+})
+pingsize:depends({
+    mode = "restart_iface"
+})
+
+forcedelay = s:option(Value, "forcedelay", translate("Forced Reboot Delay"), translate(
+                 "<i>Applies to Ping Reboot and Periodic Reboot modes</i> <br> When rebooting the router, the service will trigger a soft reboot. " ..
+                     "Entering a non-zero value here will trigger a delayed hard reboot " ..
+                     "if the soft reboot were to fail. Enter the number of seconds to wait for the soft reboot to fail " ..
+                     "or use 0 to disable the forced reboot delay."))
+forcedelay.default = "1m"
+forcedelay:depends({
+    mode = "ping_reboot"
+})
+forcedelay:depends({
+    mode = "periodic_reboot"
+})
+
+interface = s:option(ListValue, "interface", translate("Interface to Monitor"), translate(
+                "<i>Applies to Ping Reboot and Restart Interface modes</i> <br> Specify the interface to monitor and restart if a ping over it fails."))
+-- sm lifted from luci-app-wol
+for _, iface in ipairs(ifaces) do
+    if not (iface == "lo" or iface:match("^ifb.*")) then
+        local nets = net:get_interface(iface)
+        nets = nets and nets:get_networks() or {}
+        for k, v in pairs(nets) do
+            nets[k] = nets[k].sid
+        end
+        nets = table.concat(nets, ",")
+        interface:value(iface, ((#nets > 0) and "%s (%s)" % {iface, nets} or iface))
+    end
+end
+interface.rmempty = false
+interface:depends({
+    mode = "ping_reboot"
+})
+interface:depends({
+    mode = "restart_iface"
+})
+
+mmifacename = s:option(Value, "mmifacename", translate("Name of ModemManager Interface"), translate(
+                  "<i>Applies to Ping Reboot and Restart Interface modes</i> <br> If using ModemManager, you can have Watchcat restart your ModemManger interface by specifying its name."))
+mmifacename:depends({
```

Comment:
I think the depends could be one liner an does not need to be split up in 3 lines. Please have a look to the others as well in this line

Reply:
Like this?
```
pinghost:depends({ mode = "ping_reboot" })
pinghost:depends({ mode = "restart_iface" })
```